include/swift/SwiftRemoteMirror/SwiftRemoteMirrorLegacyInterop.h

```diff
@@ -354,7 +362,7 @@ swift_reflection_interop_libraryOwnsAddress(
   // Search the images list to see if the address is in one of them.
   struct SwiftReflectionInteropContextLegacyImageRangeList *Node =
     ContextRef->LegacyImageRangeList;
-  while (Node != nullptr) {
+  while (Node != NULL) {
```

Comment:
Is this a C-compatible header?

Reply:
Also, why not `!Node`?

Reply:
It is, analogous to `SwiftRemoteMirror.h`. As for why not `!Node`, you'd have to ask my brain, and it hasn't been taking questions lately.